Voycar.Api.Web/Service/Templates.cs

```diff
@@ -0,0 +1,57 @@
+namespace Voycar.Api.Web.Service;
+
+public class Templates
+{
+    public static string HtmlVerifyTemplate =
+        """
+              <!DOCTYPE html>
+                  <html lang='de'>
+                      <head>
+                          <meta charset='UTF-8'>
+                          <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+                          <title>Voycar-Konto-Verifizierung</title>
+                      </head>
+                      <body style='font-family: Arial, sans-serif; background-color: #f4f4f4; padding: 20px;'>
+                          <div
+                              style='max-width: 600px; margin: 0 auto; background-color: #ffffff; padding: 20px; border-radius: 10px; box-shadow: 0 0 10px rgba(0,0,0,0.1);'>
+                              <h1 style='color: #333333; text-align: center;'>Voycar-Konto-Verifizierung</h1>
+                              <p>Hallo {name},</p>
+                              <p>vielen Dank, dass Du Dich bei Voycar registriert hast!</p>
+                              <p>Damit wir Deine Registrierung abschließen können, musst Du Deine E-Mail-Adresse bestätigen.</p>
+                              <p>Bitte klick auf den folgenden Link, um Dein Konto zu verifizieren:</p>
```

Comment:
klein schreiben + wir haben im Frontend "Email" geschrieben 

```suggestion
                              <p>vielen Dank, dass du dich bei Voycar registriert hast!</p>
                              <p>Damit wir deine Registrierung abschließen können, musst du deine Email-Adresse bestätigen.</p>
                              <p>Bitte klicke auf den folgenden Link, um dein Konto zu verifizieren:</p>
```

Reply:
nee, aber das "Du" muss schon groß, wie in einer E-Mail oder einem Brief, oder nicht?